wear_os_bible_ereader/pubspec.yaml

```diff
@@ -14,9 +14,10 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_bloc: ^8.1.2
+  flutter_html: 3.0.0-alpha.3
```

Comment:
epub_view claims it supports versions above this, but when I tried to use latest (3.0.0-beta.1), I got these errors:

```
../../../.pub-cache/hosted/pub.dev/epub_view-3.0.0/lib/src/ui/epub_view.dart:349:13: Error: Method not found: 'tagMatcher'.

tagMatcher('img'):

^^^^^^^^^^

../../../.pub-cache/hosted/pub.dev/epub_view-3.0.0/lib/src/ui/epub_view.dart:350:17: Error: Undefined name 'CustomRender'.

CustomRender.widget(widget: (context, buildChildren) {

^^^^^^^^^^^^

../../../.pub-cache/hosted/pub.dev/epub_view-3.0.0/lib/src/ui/epub_view.dart:348:11: Error: No named parameter with the name 'customRenders'.

customRenders: {

^^^^^^^^^^^^^

../../../.pub-cache/hosted/pub.dev/flutter_html-3.0.0-beta.1/lib/flutter_html.dart:49:3: Context: Found this candidate, but the arguments don't match.

Html({

^^^^
```

Once epub_view is updated this direct dependency can be removed.